src/mlpack/core/data/imputation_methods/custom_imputation.hpp

```diff
@@ -0,0 +1,58 @@
+/**
+ * @file custom_imputation.hpp
+ * @author Keon Kim
+ *
+ * Definition and Implementation of the empty CustomImputation class.
+ */
+#ifndef MLPACK_CORE_DATA_IMPUTE_STRATEGIES_CUSTOM_IMPUTATION_HPP
+#define MLPACK_CORE_DATA_IMPUTE_STRATEGIES_CUSTOM_IMPUTATION_HPP
+
+#include <mlpack/core.hpp>
+
+using namespace std;
+
+namespace mlpack {
+namespace data {
+
+template <typename T>
+class CustomImputation
+{
+ public:
+  void Apply(const arma::Mat<T>& input,
```

Comment:
Could we put the template parameter at member function since it could deduce T automatic?